### ISSUE TITLE
How to disable arrow auto scale?

### ISSUE THREAD

**Issue by vutle** [bbecquet/Leaflet.PolylineDecorator#109]
Hi guys thanks for this great addon.

Does anyone know which option do you set to disable auto scale so that when zoom out it should appear smaller?

I'm currently use these options below, but the arrow try to autoscale and it appear bigger than the line.

 patterns: [
                    {offset: "50%", repeat: 0, symbol: L.Symbol.arrowHead({pixelSize: 15, polygon: false, pathOptions: {color: "yellow", stroke: 
                ]

![image](https://user-images.githubusercontent.com/6241928/138533872-06054822-3b42-4aa6-b63b-76ec3c21e970.png)

I want to keep it constant size.  So the arrow appear smaller when zoom out.

![image](https://user-images.githubusercontent.com/6241928/138533893-de4855b3-74d6-4c54-ace4-6eaae2fdcd4c.png)

**Comment by samuelperezdi** [comment]
Hi, I have the same problem. Did you figure something out?

**Comment by electroheadfx** [comment]
yeah me too, I want know how to do percent size relative to zoom ?
